libraries/AP_HAL_ChibiOS/hwdef/include/minimal_GPS.inc

```diff
@@ -0,0 +1,5 @@
+# include file to reduce flash by including less GPS drivers
+
+define AP_GPS_BACKEND_DEFAULT_ENABLED 0
+define AP_GPS_UBLOX_ENABLED 1
+
```

Comment:
Are you sure you don't want to include mavlink here?  It's damned useful for testing.

Reply:
it is, but very few end users will ever use it, so I decided better to have more flash space